.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- Please your `.gitignore` is missing the test.md file, this file should be created and has content of `"This file should be ignored by git"` and that file should be ignored.

Reply:
First of all, thank you for your suggestions. I just wanted to be clear on a few things:
1. Are both of the changes OPTIONAL? Since I don't see anything under the Required Changes heading, and neither I see any OPTIONAL tag with your comment. So pretty confused about that. 

2. After making the described changes, should I simply commit and push it in the "Hello-Microverse-Project" branch? Would it be merged into the same pull request or would it create a new Pull Request?

Looking forward to your reply

Reply:
Hi, the changes are not option , it there were you would see [Optional]. 

You can make the changes requested and then request for another review. 


If you have other questions please feel free to ask me